### ISSUE TITLE
artifacts_management_1 cant access management.json

### ISSUE THREAD

**Issue by ghost** [netbirdio/netbird#1429]
Using latest self hosted version. Running on up to date fedora coreos. After generating all files and doing docker-compose up I get following docker log entry for management container:

Error: failed reading provided config file: /etc/netbird/management.json: open /etc/netbird/management.json: permission denied

everything seems to be ok and for the life of me I cant figure it out

**Comment by mlsmaycon** [comment]
Hello @jaanuskoiv, can you provide the folder structure and from which place you are running the docker-compose commands?

from the infreastructure_files you can run:
```shell
ls -lR
```

**Comment by jaanuskoiv** [comment]
root@netbird01:/opt/netbird/infrastructure_files# ls -lR
.:
total 100
drwxr-xr-x. 2 root root  4096 Jan  2 10:40 artifacts
-rw-r--r--. 1 root root  4465 Jan  2 08:41 base.setup.env
-rwxr-xr-x. 1 root root  8110 Jan  2 08:41 configure.sh
-rw-r--r--. 1 root root  2674 Jan  2 08:41 docker-compose.yml.tmpl
-rw-r--r--. 1 root root  3762 Jan  2 08:41 docker-compose.yml.tmpl.traefik
-rw-r--r--. 1 root root 23815 Jan  2 08:41 getting-started-with-zitadel.sh
-rw-r--r--. 1 root root  3239 Jan  2 08:41 management.json.tmpl
-rw-r--r--. 1 root root  2125 Jan  2 08:41 nginx.tmpl.conf
-rw-r--r--. 1 root root  4335 Jan  2 09:11 setup.env
-rw-r--r--. 1 root root  3843 Jan  2 08:41 setup.env.example
drwxr-xr-x. 2 root root    23 Jan  2 08:41 tests
-rw-r--r--. 1 root root 26579 Jan  2 08:41 turnserver.conf.tmpl

./artifacts:
total 112
-rw-r--r--. 1 root root  2467 Jan  2 10:40 docker-compose.yml
-rw-r--r--. 1 root root  2463 Jan  2 09:12 docker-compose.yml.bkp.1704186724
-rw-r--r--. 1 root root  2467 Jan  2 10:40 docker-compose.yml.bkp.1704192013
-rw-r--r--. 1 root root  3308 Jan  2 10:40 management.json
-rw-r--r--. 1 root root  3308 Jan  2 09:12 management.json.bkp.1704186724
-rw-r--r--. 1 root root  3308 Jan  2 10:40 management.json.bkp.1704192013
-rw-r--r--. 1 root root  1753 Jan  2 10:40 openid-configuration.json
-rw-r--r--. 1 root root 26584 Jan  2 10:40 turnserver.conf
-rw-r--r--. 1 root root 26584 Jan  2 09:12 turnserver.conf.bkp.1704186724
-rw-r--r--. 1 root root 26584 Jan  2 10:40 turnserver.conf.bkp.1704192013

./tests:
total 4
-rw-r--r--. 1 root root 1282 Jan  2 08:41 setup.env

**Comment by jaanuskoiv** [comment]
also forgot to mention that iam running docker-compose command inside that artifacts directory. full path is /opt/netbird/infrastructure_files/artifacts

**Comment by jaanuskoiv** [comment]
docker-compose has following regarding management container:

  # Management
  management:
    image: netbirdio/management:latest
    restart: unless-stopped
    depends_on:
      - dashboard
    volumes:
      - netbird-mgmt:/var/lib/netbird
      - netbird-letsencrypt:/etc/letsencrypt:ro
      - ./management.json:/etc/netbird/management.json
    ports:
      - 33073:443 #API port
  #    # command for Let's Encrypt validation without dashboard container
  #    command: ["--letsencrypt-domain", "cc.bisly.ai", "--log-file", "console"]
    command: [
      "--port", "443",
      "--log-file", "console",
      "--disable-anonymous-metrics=true",
      "--single-account-mode-domain=cc.bisly.ai",
      "--dns-domain=netbird.selfhosted"
      ]

**Comment by jaanuskoiv** [comment]
Installed on ubuntu 23.04 and same issue is not present so it must be fedora coreos specific issue

**Comment by mlsmaycon** [comment]
thanks, @jaanuskoiv. Is SELinux enabled on the core node?

**Comment by jaanuskoiv** [comment]
yes and that is the reason most likely. completely forgot about that

**Comment by PizzaLovingNerd** [comment]
Closing this as stale, the issue author is deleted, and this is likely an SELinux issue rather than a NetBird bug.
